app/src/main/java/ch/epfl/balelecbud/RegisterUserActivity.java

```diff
@@ -40,6 +39,7 @@ public void accept(Void aVoid, Throwable throwable) {
                 if (throwable != null) {
                     Toast.makeText(RegisterUserActivity.this, throwable.getCause().getLocalizedMessage() ,Toast.LENGTH_SHORT).show();
                 } else {
+                    Log.d("HERE","Auth complete");
```

Comment:
You can put a more meaningful TAG

Reply:
You're right, this should've been deleted. My bad